end_area: 270
end_frontier: 0
end_time: 10000
end_area_diff: 20
end_area_diff_interval: 120
end_area_diff_rate: 0.3
end_area_diff_averate: 0.3
